dataland-frontend/tests/e2e/specs/quality-assurance/CheckQaProcedure.ts

```diff
@@ -35,41 +29,14 @@ describeIf(
         DataTypeEnum.EutaxonomyFinancials,
         testCompany,
         testData.t,
-        fillEuTaxonomyForm,
+        (data) => fillAndValidateEuTaxonomyCreditInstitutionForm(data),
         (req) => (req.headers["REQUIRE-QA"] = "true"),
```

Comment:
This code caught my attentation.
I just talked to Florian and it seems that the code block here accidentally made its way to main.
We should not upload a dataset via the form here. We should rather use the api, which is much much faster.
So I agreed with Florian that the test shall be smth. like:

- upload company and eu taxo dataset via api
- run `testSubmittedDatasetIsInReviewList(...)`

Reply:
👍